lib/core/lib/generators/ros/model/model_generator.rb

```diff
@@ -7,12 +7,29 @@ class ModelGenerator < Rails::Generators::NamedBase
     include GeneratorsHelper
 
     def create_files
-      create_file "app/models/#{name}.rb", <<~FILE
-        # frozen_string_literal: true
+      invoke(:model)
 
-        class Entity < #{parent_module}ApplicationRecord
-        end
-      FILE
+      gsub_created_file
```

Comment:
Make the last two methods public and they will be automatically invoked so you don't need to call them at all